PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -65,11 +70,28 @@ if (isNullOrWhitespace(searchFolder)) {
     searchFolder = tl.getVariable('System.DefaultWorkingDirectory');
 }
 
+let dtpBaseUrl : string;
+let dtpUsername : string;
+let dtpPassword : string;
 const localSettings = loadSettings(localSettingsPath);
 if (localSettings) {
     tl.debug('dtp.url: ' + localSettings['dtp.url']);
     tl.debug('dtp.user: ' + localSettings['dtp.user']);
     tl.debug('dtp.password: ' + localSettings['dtp.password']);
+    tl.debug('dtp.server: ' + localSettings['dtp.server']);
+    tl.debug('dtp.port: ' + localSettings['dtp.port']);
+    tl.debug('dtp.context.path: ' + localSettings['dtp.context.path']);
+
+    dtpBaseUrl = getDtpBaseUrl(localSettings);
+    tl.debug('dtpBaseUrl: ' + dtpBaseUrl);
+    dtpUsername = localSettings['dtp.user'];
+    if (isNullOrWhitespace(dtpUsername)) {
+        tl.warning('The username for DTP server authentication is not specified.');
```

Comment:
Instead of using the phrase "The username for DTP server", let's use the property name which would give better guidance for user.